src/views/HomeView.vue

```diff
@@ -1,21 +1,66 @@
 <script setup>
-import { ref, watchEffect } from "vue";
 import Home from "../components/Home.vue";
+import UserContributionForm from "../components/UserContributionForm.vue";
+import { onMounted, ref, watchEffect } from "vue";
 import { getTasks } from "../stores/api.js";
 
 const tasks = ref([]);
-const isError = ref(false)
+const isError = ref(false);
+const currentTaskIndex = ref(0);
+const currentTask = ref(null);
 
-watchEffect(async () => {
+const props = defineProps({
+  currentUser: String,
+});
+
+onMounted(async () => {
   try {
-    tasks.value = await getTasks()
-  } catch(error) {
-    isError.value = true
-    console.log(error)
+    tasks.value = await getTasks();
+    currentTask.value = tasks.value[currentTaskIndex.value];
```

Comment:
Looks like this is repeated on line 26, makes me wonder why we had to change the watchEffect to onMounted?

Reply:
You're right, this line was unnecessary.

Reply:
Removed in [b3895cc](https://github.com/wikimedia/toolhunt-ui/pull/41/commits/b3895ccc14df0fc0d4bc2bec6257fa73d9fa4ecb)